src/Diagnostics/Diagnostics.jl

```diff
@@ -38,34 +38,38 @@ using ..Writers
 using CLIMAParameters
 using CLIMAParameters.Planet: planet_radius
 struct EarthParameterSet <: AbstractEarthParameterSet end
```

Comment:
Please remove this (sorry, I missed stopping this from going in)

Reply:
This was already there, I've just started the process of cleaning up.